libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -451,8 +444,7 @@ float AP_Baro::get_EAS2TAS(void)
     if (!is_positive(eas2tas_squared)) {
         return 1.0f;
     }
-    _EAS2TAS = sqrtf(eas2tas_squared);
-    _last_altitude_EAS2TAS = altitude;
+    const float _EAS2TAS = sqrtf(eas2tas_squared);
     return _EAS2TAS;
```

Comment:
```suggestion
    return sqrtf(eas2tas_squared);
```

Reply:
Ha! Yeah, I definitely should have just done that. Fixed.